server/tests/secrets/test_secret_ref_utils.py

```
@@ -0,0 +1,320 @@
+        monkeypatch.setattr(sru, "_resolve_org", MagicMock(return_value="org-7"))
+        return cursor
+
+    @pytest.mark.parametrize("raw_provider", ["GCP", "Gcp", "gCp", "GcP"])
+    def test_has_user_credentials_passes_lowercase_to_sql(
+        self, monkeypatch, raw_provider,
+    ):
+        cursor = self._make_db(monkeypatch, (1,))
+        manager = SecretRefManager()
+        manager.has_user_credentials("u-1", raw_provider)
+
+        executed_sql, params = cursor.execute.call_args.args
```

Ruff (0.15.12)

[warning] Unpacked variable `executed_sql` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)